src/definitions.jl

```diff
@@ -561,3 +576,58 @@ Pre-plan an optimized real-input unnormalized transform, similar to
 the same as for [`brfft`](@ref).
 """
 plan_brfft
+
+##############################################################################
+
+region(p::Plan) = p.region
+region(p::ScaledPlan) = region(p.p)
+
+# Projection style (:none, :real, or :real_inv) to handle real FFTs
+function projection_style end
+# Length of halved dimension, needed only for irfft 
+function irfft_dim end
```

Comment:
Is this actually needed if it's only used for `irfft`?

Reply:
I think so. Because an `AdjointPlan` of RFFT needs to know the original length of the halved dimension, as the adjoint plan acts on an array where the dimension has already been halved and it cannot figure the original length without help -- `RealInverseProjectionStyle` is not enough to specify it as there are two options.

Reply:
Is it possible to include it in `RealInverseProjectionStyle` by eg adding a field?